Meniga/Meniga-ios-sdk/External/Sync/MNFSynchronization.h

```diff
@@ -78,7 +88,25 @@ NS_ASSUME_NONNULL_BEGIN
  */
 +(MNFJob*)startSynchronizationWithWaitTime:(NSNumber *)waitTime completion:(nullable MNFSynchronizationCompletionHandler)completion;
 
-+(MNFJob*)startSynchronizationForRealmUserWithId:(NSNumber*)realmUserId waitTime:(NSNumber*)waitTime completion:(nullable MNFSynchronizationCompletionHandler)completion;
+/**
+ @abstract Starts the accounts synchronization process for a specific realm with a session token already provided.
+
+ @param realmUserId  The id of the realm account user - a realm is a "department" in e.g. a bank. Most organizations (most often banks) have only one but some
+ large organizations can have many realms (e.g. insurance, banking, credit cards and so on). This id identifies the user's realm user account.
+ @param sessionToken A token from a previous (e.g. authentication) operation that should be carried over to the sync process
+ @param waitTime The amount of time the background process should try to check if the sync has completed before terminating.
```

Comment:
We can have an info about time unit here.

Reply:
Added